docs/UserGuide.md

```diff
@@ -493,90 +445,90 @@ Example:
 
-The `match auto` command instructs PropertyWhiz to automatically match buyers to properties.
+![Match auto](images/MatchAutoUi.png)
 
-Upon entering `match auto`, PropertyWhiz will intelligently pair properties with buyers based on their compatibility. The matches will then be displayed to you starting with the most compatible pairings at the top.
+Format: `match auto`
+
+Upon entering `match auto`, PropertyWhiz will intelligently pair the displayed properties and buyers based on their compatibility. The matches will then be displayed to you starting with the most compatible pairings at the top.
 
 PropertyWhiz determines compatibility based on the number of tags in common, the buyer's budget, as well as the property price. Matches where the buyer and property have more tags in common are considered more compatible. Likewise for matches where the property price is within the buyer's budget.
 
 After running `match auto`, enter `back` into the command box to return to the previously shown list of buyers and properties.
 
-### Importing data from csv file : `import`
+### Importing data from CSV file : `import`
 
-Imports buyers or properties from csv file. Imported items will be added to the front of the **currently displayed list**.
+Use this command to import buyers or properties from a CSV file. Imported items will be added to the front of the list.
 
 Format: `import (buyer | property)`
 
 * You can select the import file location from a pop-up dialog box.
-* The buyer or property list imported must be [valid](#valid-propertiesbuyers). Some criteria are as follows:
+* The buyer or property list imported must be valid. Some criteria are as follows:
   * No duplicates
-  * Valid fields e.g. name, email etc.
+  * All fields [valid](#valid-propertiesbuyers)
   * No missing fields
-* The csv must begin with a header which includes the names of each column.
+* The CSV must begin with a header that includes the names of each column.
 
-See [below](#exporting-data-to-csv-file--export) for example csv files.
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+
+If you are using Microsoft Excel, you can easily export your data to a CSV file and import it to PropertyWhiz!
+
+</div>
 
-### Exporting data to csv file : `export`
+See [below](#exporting-data-to-csv-file--export) for example CSV files.
 
-Exports buyers or properties in the **currently displayed list** to csv file.
+### Exporting data to CSV file : `export`
+
+Use this command to export currently displayed buyers or properties to a CSV file.
 
 Format: `export (buyer | property)`
 
 * You can select the export file location from a pop-up dialog box.
 
-Example csv output of  `export property`
+Example CSV output of  `export property`
 <a name="csvExample"> </a>
```

Comment:
This link can be deleted now that we are linking to `Exporting data to CSV file` instead.